hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,464 @@
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Aug 11 14:09:51 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_aes_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_hmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_kmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_otbn_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_io_meas.u_meas
+ANNOTATION: "VC_COV_UNR"
+Block 17 "2798412932" ";"
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_io_div2_meas.u_meas
```

Comment:
are the measured instances blackboxed..?

Reply:
No, The exclusion is only for a certain block 17 within the instance.

Reply:
hmm...but what does that mean...? block 17...

Reply:
Okay, I am regenerating the coverage database and will find out... It probably refers to a basic block number ("basic block" is compiler jargon).

Reply:
Hmm, Notice it says
  Block 17 "2798412932" ";"
And the only basic block with a single ";" statement is in line 130, at
  default:;
which is definitely not covered.

Reply:
ahh, okay that makes sense. thank you.